matter_server/server/device_controller.py

```diff
@@ -819,6 +866,28 @@ async def get_node_ip_addresses(
         self._last_known_ip_addresses[node_id] = ip_adresses
         return ip_adresses if scoped else [x.split("%")[0] for x in ip_adresses]
 
+    @api_command(APICommand.IMPORT_TEST_NODE)
+    async def import_test_node(self, dump: str) -> None:
+        """Import test node(s) from a HA or Matter server diagnostics dump."""
+        try:
+            dump_data = cast(dict, json_loads(dump))
+        except JSON_DECODE_EXCEPTIONS as err:
+            raise InvalidArguments("Invalid json") from err
+        # dump can either be a single dump or a full dump with multiple nodes
+        dump_nodes: list[dict[str, Any]]
+        if "node" in dump_data["data"]:
```

Comment:
It would be good to document the dump schema that needs to be followed here.

Reply:
yeah its HA diagnostics but I figured we might also want to create an export from the dashboard itself

Reply:
I have added a comment here about the format.
In a follow-up PR I'll add an export button in the dashboard as well to dump the diagnostics.

Reply:
changed to debug (and opened PR to add verbose debug level)